src/main/puzzle/scenario-library.gd

```diff
@@ -0,0 +1,106 @@
+extends Node
```

Comment:
This is also worth mentioning in the commit comment: 'Extracted scenario-library.gd for storing puzzle scenarios.'

Reply:
Commit comment now mentions `Extracted scenario-library.gd
from scenario-menu.gd`